pgjdbc/src/main/java/org/postgresql/jdbc/PgResultSet.java

```diff
@@ -2098,9 +2098,6 @@ public long getLong(int columnIndex) throws SQLException {
     if (isBinary(columnIndex)) {
       int col = columnIndex - 1;
       int oid = fields[col].getOID();
-      if (oid == Oid.INT8) {
-        return ByteConverter.int8(this_row[col], 0);
```

Comment:
`getLong` is likely to be used for `INT8`
I don't think they will use `getLong` for strings and stuff like that often